libraries/AP_HAL_ChibiOS/hwdef/include/minimize_common.inc

```diff
@@ -0,0 +1,113 @@
+# for now we make this define, but we should migrate from this to much
+#  more specific defines - added below.
+define HAL_MINIMIZE_FEATURES 1
+
+# disable emitting nice strings when activating RC channel aux functions:
+define AP_RC_CHANNEL_AUX_FUNCTION_STRINGS_ENABLED 0
+
+# low-end boards aren't expected to be used in environments where
+# things like satellite communications are required:
+define HAL_HIGH_LATENCY2_ENABLED 0
+
+# Crossfire telemetry must be explicitly enabled on minimized boards:
+define HAL_CRSF_TELEM_ENABLED 0
```

Comment:
strongly disagree

Reply:
Express LRS (using Crossfire) is becoming almost standard among hobbyists. I agree with @Hwurzburg - not a good idea to disable this by default.

Reply:
> strongly disagree

Sure.  Not trying to fix everything in this PR ATM.

Perhaps we should, but I'd like comment on the structure before I try to make *everyone* happy with it....